cmd/server/config.go

```diff
@@ -61,6 +63,10 @@ func (c *config) ApplyDefaults() {
 		c.RPCEndpoint = "http://localhost:9650"
 	}
 
+	if c.IndexerEndpoint == "" {
+		c.IndexerEndpoint = c.RPCEndpoint
```

Comment:
Does this work? Our index apis have a specific endpoint which is distinct from our rpc endpoints (ref: https://docs.avax.network/apis/avalanchego/apis/index-api#endpoints)

Reply:
see below